docs-ui/1.0.0/security_critical_api_specs.yaml

```diff
@@ -345,27 +345,34 @@ paths:
                           last_validated_child_block_timestamp:
                             type: integer
                             format: int64
+                            description: The time of the last Watcher sync with the child chain.
                           last_validated_child_block_number:
                             type: integer
                             format: int64
+                            description: The last block that the Watcher has synced with the child chain.
```

Comment:
Perhaps specify _child chain_ block in the descriptions for the child chain ones?

Reply:
done